source/includes/fact-connect-srv-description.rst

```diff
@@ -0,0 +1,28 @@
+*New in Compass version 1.12*
+     
+This toggle indicates whether the provided :guilabel:`Hostname`
+is an
+:manual:`SRV Record </reference/connection-string/#dns-seedlist-connection-format>`.
```

Comment:
Do we need to say anything about SRV connections string being for MongoDB versions 3.6 and later?

Reply:
I don't see the sentence?

Reply:
Oops I think I had in it a prior commit and then lost it. Re-added in a note but I don't love it.

Reply:
I see it - looks good 👍